docs/sdk/pubsub/fragments/android/working-api.md

```diff
@@ -46,6 +46,47 @@ try {
 }
 ```
 
+### Custom Authentication
```

Comment:
This is part of the Android section and doesn't appear on the [current docs site](https://aws-amplify.github.io/docs/sdk/android/pubsub#pubsub). @desokroshan Can you verify this documentation?

Reply:
https://aws-amplify.github.io/docs/sdk/android/pubsub#custom-authentication